.github/workflows/ci.yaml

```diff
@@ -0,0 +1,35 @@
+name: Terraform CI
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  Terraform-Lint-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: "1.5.7"
+
+      - name: Terraform fmt
```

Comment:
Can you just add this lint check to cloudbuild.yaml and remove the github actions workflow? I don't think we're at a point where we want to manage both cloudbuild and github actions for presubmit.

Reply:
(also because cloudbuild is a blocking pre-submit now)